pkg/provider/provider.go

```diff
@@ -136,3 +137,13 @@ func MergeDeploymentVars(ms ...map[string]string) map[string]string {
 	}
 	return res
 }
+
+// GetDefaultDeploymentVars shows default deployment variables.
+func (dr *DeploymentResource) GetDefaultDeploymentVars(parseContext *kingpin.ParseContext) error {
+	fmt.Print("-------------------\n   DeploymentVars   \n------------------- \n")
+	for key, value := range dr.DefaultDeploymentVars {
```

Comment:
This will only show default deploymentvars, we want to see all the deployment vars that'll be passed when parsing the manifest files. Therefore this function should be inside the specific provider code.(in our case `gke.go`) 

So we want to see `FlagDeploymentVars`+`DefaultDeploymentVars`+if we set any provider specific vars inside `SetupDeploymentResources`

We'd want to get info about:
https://github.com/prometheus/test-infra/blob/623686db4f315d316d6ba3bce3c58dc288b14c6c/pkg/provider/gke/gke.go#L67

https://github.com/prometheus/test-infra/blob/623686db4f315d316d6ba3bce3c58dc288b14c6c/pkg/provider/gke/gke.go#L133

If there are no FlagDeploymentvars or provider specific  deployment it'll simply show the defaultdeploymentvars :)

Reply:
Done